web-server/.gitignore

```diff
@@ -0,0 +1,4 @@
+### Rust ###
+# Generated by Cargo
+# will have compiled files and executables
+target
```

Comment:
I am not 100% on the behavior of .gitignore files in sub-directories. Will this correctly ignore `web-server/target`?